.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,36 @@
+# Based on https://github.com/actions/starter-workflows/blob/main/ci/python-publish.yml
+name: Publish to Pypi
+
+on:
+  release:
+    types: [created]
+
+  push:
+    branches:
+      - '*/fixed-publishing-to-pypi'
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: "__token__"
+        TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+      run: |
+        python setup.py sdist
```

Comment:
Personally, I've been doing `python setup.py bdist_wheel sdist` which gives us a wheel too. I don't really know what the practical difference is.

As you can tell, I've only been putting releases up on PyPI, not labeling/branching anything in github, so I might need your help with that too :)

Reply:
Aren't wheels useful only when you have compiled binaries included in the package?

Re releases: here is the [relevant GitHub doc page](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository).

I usually create a separate commit (such as "Release 1.2.3") which contains the change log in the message body (as bullet list). Then I create the release with the tag "v1.2.3" and the title of the release is "CrossHair 1.2.3". I repeat the changelog in the release description to make it easier for the users to find it. And that's it :).

Reply:
Thanks!
And, yeah, maybe the wheel [installs a little bit faster](https://stackoverflow.com/questions/59619940/what-is-the-point-of-built-distributions-for-pure-python-packages)? Low stakes either way I suppose.

Reply:
Oh, I didn't know that, thanks! Let me add bdist then before you merge.